core_api/src/app.py

```diff
@@ -47,6 +48,30 @@
 storage_handler = ElasticsearchStorageHandler(es_client=es, root_index="redbox-data")
 
 
+mime_type_to_content_type = {
```

Comment:
one to discuss with @rachaelcodes

Reply:
If we are going down this route, we might need to include multiple options for some content types, as browsers can be inconsistent with MIME types (see https://github.com/getgrav/grav/issues/3422 for a discussion of this from PHP-land). Though apparently it's not unheard of for .eml files to be assigned 'application/octet-stream' https://stackoverflow.com/questions/65698647/what-is-the-mime-type-of-eml-files-in-emails which really wouldn't be helpful at all. 

At the very least we should probably add: "text/xml": "xml" https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types . Alternatively, we could take the file extension instead (which is what I chose to do for REDBOX-71)

Given that we should be double-checking user submitted MIME types etc anyway, I think in the future we should move towards a system where we get the MIME types assessed by a service like ClamAV, which should hopefully give us a more standard list (and maybe even the file type).

Reply:
The other thing about REDBOX-71: we're also looking to pass the file type directly to core-api, so this file check wouldn't be happening in core-api any more. https://github.com/i-dot-ai/redbox-copilot/pull/157/files#diff-ba6d55b32fa5e31957b2d97cbe5c55d37989f3b72bd66db3653ef54d9db3f52fR106

Reply:
what process uses this field in a meaningful way? could we either:
* make it a plain `str`
* drop it all together